hyperspy/tests/io/test_bcf.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import numpy as np
+
+from nose.tools import assert_true
+from hyperspy.io import load
+
+test_files = ['P45_instructively_packed_16bit_compressed.bcf',
+              'P45_12bit_packed_8bit.bcf',
+              'P45_the_default_job.bcf']  # for not compressed testing
+
+
+def test_load():
+    # test bcf from hyperspy load function level
+    # some of functions can be not covered
+    # it cant use cython parsing implementation, as it is not compiled
+    for thingy in test_files:
+        my_path = os.path.dirname(__file__)
+        filename = os.path.join(my_path, 'bcf_data', thingy)
+        print('testing bcf loading from hyperspy level...')
+        s = load(filename, downsample=2, cutoff_at_kV=10)
+        s = load(filename)
```

Comment:
Yes, separate into more functions, to explicitly test that the downsampling and cutoff works (both independently).

What I mean is, for example, this:
```python
In [13]: s = hs.load('tests/io/bcf_data/P45_the_default_job.bcf')
spectrum have no dead time records...

In [14]: s
Out[14]: 
[<Image, title: BSE, dimensions: (|100, 75)>,
 <Image, title: SE, dimensions: (|100, 75)>,
 <EDSSEMSpectrum, title: EDX, dimensions: (100, 75|2048)>]

In [15]: np.where(s[-1].inav[37,37].data)                           
Out[15]: 
(array([  45,   46,   48,   49,   50,   52,  100,  102,  103,  155,  183,
         193,  218,  220,  223,  227, 1321]),)

In [16]: locations = np.where(s[-1].inav[37,37].data)[0]

In [17]: s[-1].inav[37,37].data[locations]
Out[17]: array([1, 1, 1, 1, 1, 1, 2, 1, 1, 1, 1, 1, 3, 1, 1, 1, 1], dtype=uint8)

In [18]: values = s[-1].inav[37,37].data[locations]
```

I just randomly picked the (37,37) pixel, obviously.

You can then define the locations and values manually and check that they match (and that all other elements are zero!)

```python
values = np.array([1, 1, 1, 1, 1, 1, 2, 1, 1, 1, 1, 1, 3, 1, 1, 1, 1], dtype=uint8)
locations = np.array([  45,   46,   48,   49,   50,   52,  100,  102,  103,  155,  183, 193,  218,  220,  223,  227, 1321])

```

So no new numpy files are necessary.